DMCompiler/DM/Visitors/DMProcBuilder.cs

```diff
@@ -161,9 +164,19 @@ public void ProcessStatementSet(DMASTProcStatementSet statementSet)
         {
             var attribute = statementSet.Attribute.ToLower();
             // TODO deal with "src"
-            if (!DMExpression.TryConstant(_dmObject, _proc, statementSet.Value, out var constant) && attribute != "src") {
+            if(attribute == "src")
+            {
+                DMCompiler.UnimplementedWarning(statementSet.Location, "'set src' is unimplemented");
+                return;
+            }
+            if (!DMExpression.TryConstant(_dmObject, _proc, statementSet.Value, out var constant)) {
                 throw new CompileErrorException(statementSet.Location, $"{attribute} attribute should be a constant");
             }
+            if (statementSet.WasInKeyword)  // check if it was 'set x in y' or whatever
+            {                               // (which is illegal for everything except setting src to something)
+                DMCompiler.Error(new CompilerError(statementSet.Location, "Use of 'in' keyword is illegal here. Did you mean '='?"));
+                //fallthrough into normal behaviour because this error is kinda pedantic
```

Comment:
It's grammatically incorrect to use `in` there, BYOND does not accept it, it's just not what DM is. By "pedantic" I mean that, this error shouldn't really confuse the compiler too much since it's very likely the user meant to use `=` here.